Kernel/Modules/AdminTranslations.pm

```diff
@@ -2,7 +2,7 @@
 # OTOBO is a web-based ticketing system for service organisations.
 # --
 # Copyright (C) 2001-2020 OTRS AG, https://otrs.com/
-# Copyright (C) 2019-2024 Rother OSS GmbH, https://otobo.io/
+# Copyright (C) 2019-2024 Rother OSS GmbH, https://otobo.de/
```

Comment:
Looks like CodePolicy is not up to date, maybe you need to check out the rel-11_0 branch